src/routes/+layout.svelte

```diff
@@ -1,5 +1,31 @@
-<script>
+<script lang="ts">
   import '../app.css';
+  import Navbar from '$lib/layout/Navbar.svelte';
+  import Footer from '$lib/layout/Footer.svelte';
+  import Sidebar from '$lib/layout/Sidebar.svelte';
 </script>
 
-<slot />
+<!-- Uncomment <Sidebar /> and comment main + navbar + footer to correctly see the sidebar  -->
```

Comment:
Is this comment needed?

Reply:
This was still here from a rebase conflict i believe :P